CONTRIBUTORS.md

```diff
@@ -97,13 +111,17 @@ The following individuals have contributed code to Galaxy:
 * David Trudgian <dave@trudgian.net> <david.trudgian@utsouthwestern.edu>
 * Nitesh Turaga <nitesh.turaga@gmail.com>
 * Clayton Turner <clayclay911@gmail.com>
+* Jesse c j van Dam <jesse.vandam@wur.nl>
```

Comment:
It was in the name as he signed in his commits.  Let's go with this, and if he'd like to change it that's really easy to do later.